---
layout: archive
title: "Curriculum Vitae"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

*Last updated: Jun 3, 2024*

*Please find an extended version of my CV in this **<a href="https://drive.google.com/file/d/1AtVbptyhfBHzxN-v-K8bItVa-xgiDXxv/view?usp=sharing" target="_blank">link</a>**.*

## 1. Education

* **Ph.D** in "Innovative technologies and sustainable use of Mediterranean Sea fishery and biological resources" (FishMed-PhD), University of Bologna, 2025 (expected)\
Subject: *Comparative genomics of bivalve sex-determination related genes*\
Supervisor: *prof. Andrea Luchetti*
* **M.Sc** in "Biodiversity and evolution", University of Bologna, 2021\
Thesis: *Identification of* Hox *and* ParaHox *genes in tadpole shrimp genomes (Pancrustacea, Branchiopoda, Notostraca)*\
Supervisor: *prof. Andrea Luchetti*
* **B.Sc** in "Biological Sciences", University of Bologna, 2019\
Thesis: *Species delimitation and phylogenetic analysis of the Australian genus* Candovia *(Insecta, Phasmida)*\
Supervisor: *prof. Barbara Mantovani*

## 2. Publications

* <ins>Nicolini F</ins>, Ghiselli F, Luchetti A, & Milani L. (2023). **Bivalves as emerging model systems to study the mechanisms and evolution of sex determination: a genomic point of view**. *Genome Biology and Evolution*, *15*(10), evad181. doi: [10.1093/gbe/evad181](https://doi.org/10.1093/gbe/evad181)

* Martelossi J, <ins>Nicolini F</ins>, Subacchi S, Pasquale D, Ghiselli F, & Luchetti A. (2023). **Multiple and diversified transposon lineages contribute to early and recent bivalve genome evolution**. *BMC Biology*, *21*(1), 1-23. doi: [10.1186/s12915-023-01632-z](https://doi.org/10.1186/s12915-023-01632-z)

* <ins>Nicolini F</ins>, Martelossi J, Forni G, Savojardo C, Mantovani B, & Luchetti A. (2023). **Comparative genomics of *Hox* and *ParaHox* genes among major lineages of Branchiopoda with emphasis on tadpole shrimps**. *Frontiers in Ecology and Evolution*, 11, 23. doi: [10.3389/fevo.2023.1046960](https://doi.org/10.3389/fevo.2023.1046960)

* Forni G, Cussigh A, Brock PD, Jones BR, <ins>Nicolini F</ins>, Martelossi J, ... & Mantovani B. (2023). **Taxonomic revision of the Australian stick insect genus *Candovia* (Phasmida: Necrosciinae): insight from molecular systematics and species-delimitation approaches**. *Zoological Journal of the Linnean Society*, *197*(1), 189-210. doi: [10.1093/zoolinnean/zlac074](https://doi.org/10.1093/zoolinnean/zlac074)

## 3. Outreach activity

*For a more detailed description of my outreach commitment, please have a look also at the [Outreach](/outreach/) section in this website.*

* Editor and web writer, BioPills\
  *Jul 2017 – ongoing*

* BiGeA Day 2023, University of Bologna\
  *May 27, 2023*

* University of Bologna opening days\
  *Nov 18, 2022*

## 4. Professional skills

### 4.1. Bioinformatics

<ul>
<li><b>Programming and markup languages</b></li>
<details>
  <summary><i>More</i></summary>
  <small>(beginner ▯▯▯▯▯ advantaced)</small>
  <ul>
    <li><code>bash</code></li>
      ▮▮▮▮▯
    <li><code>R</code></li>
      ▮▮▮▯▯
    <li><code>python</code></li>
      ▮▮▯▯▯
    <li>LaTeX</li>
      ▮▮▯▯▯
    <li><code>html</code></li>
      ▮▯▯▯▯
  </ul>
</details>
<br>

<li><b>Phylogenetics</b></li>
<details>
  <summary><i>More</i></summary>
  <ul>
    <li>ML phylogenetic analysis (<a href="https://github.com/iqtree/iqtree2" target="_blank"><code>IQTREE</code></a>, <a href="https://github.com/amkozlov/raxml-ng" target="_blank"><code>RAxML-NG</code></a>, <a href="http://www.microbesonline.org/fasttree/" target="_blank"><code>FastTree</code></a>)</li>
    <li>Bayesian phylogenetic analysis (<a href="https://nbisweden.github.io/MrBayes/" target="_blank"><code>MrBayes</code></a>, <a href="http://www.atgc-montpellier.fr/phylobayes/" target="_blank"><code>PhyloBayes</code></a>)</li>
    <li>Global alignment tools (<a href="https://mafft.cbrc.jp/alignment/software/" target="_blank"><code>MAFFT</code></a>, <a href="https://github.com/rcedgar/muscle" target="_blank"><code>MUSCLE</code></a>, <a href="http://www.clustal.org/omega/" target="_blank"><code>Clustal Omega</code></a>, <a href="http://trimal.cgenomics.org/" target="_blank"><code>trimAl</code></a>, <a href="https://home.cc.umanitoba.ca/~psgendb/doc/Castresana/Gblocks_documentation.html" target="_blank"><code>GBlocks</code></a>)</li>
    <li>Tree calibration (<a href="https://github.com/tothuhien/lsd2" target="_blank"><code>lsd2</code></a>, <a href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank"><code>MCMCTree</code></a>)</li>
    <li>Tree decomposition (<a href="https://github.com/JSdoubleL/DISCO" target="_blank"><code>DISCO</code></a>)</li>
  </ul>
</details>
<br>


<li><b>Comparative genomics</b></li>
<details>
  <summary><i>More</i></summary>
  <ul>
  <li>Orthology inference (<a href="https://github.com/davidemms/OrthoFinder" target="_blank"><code>OrthoFinder</code></a>, <a href="https://github.com/xgrau/possvm-orthology" target="_blank"><code>Possvm</code></a>, <a href="https://github.com/hillerlab/TOGA" target="_blank"><code>TOGA</code></a>)</li>
  <li>Gene annotation (<a href="https://github.com/ncbi/blast_plus_docs" target="_blank"><code>BLAST</code></a>, <a href="https://github.com/bbuchfink/diamond" target="_blank"><code>DIAMOND</code></a>, <a href="https://github.com/EddyRivasLab/hmmer" target="_blank"><code>HMMER</code></a>, <a href="https://github.com/ebi-pf-team/interproscan" target="_blank"><code>InterProScan</code></a>, <a href="https://github.com/UCSC-LoweLab/tRNAscan-SE" target="_blank"><code>tRNAscan-SE</code></a>)</li>
  <li>Genome quality assessment (<a href="https://busco.ezlab.org/" target="_blank"><code>BUSCO</code></a>)</li>
  <li>Molecular evolution (<a href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank"><code>PAML</code></a>, <a href="http://hyphy.org/" target="_blank"><code>HyPhy</code></a>, <a href="https://github.com/for-giobbe/BASE" target="_blank"><code>BASE</code></a>, <a href="https://github.com/mrrlab/RRTree" target="_blank"><code>RRTree</code></a>)</li>
  <li>Gene family evolution (<a href="https://github.com/hahnlab/CAFE5" target="_blank"><code>CAFE5</code></a>, <a href="https://github.com/BenoitMorel/GeneRax" target="_blank"><code>GeneRax</code></a>)
  <li>Gene macro-synteny (<a href="https://github.com/wyp1125/MCScanX" target="_blank"><code>MCScanX</code></a>/<a href="https://github.com/tanghaibao/jcvi" target="_blank"><code>JCVI</code></a>)</li>
  <li>Detection of sex-related loci (<a href="https://gitlab.in2p3.fr/sex-det-family/sex-detector" target="_blank"><code>SEX-DETector</code></a>, <a href="https://gitlab.in2p3.fr/sex-det-family/sdpop" target="_blank"><code>SDpop</code></a>)</li>
  <li>Other (<a href="https://github.com/NBISweden/AGAT" target="_blank"><code>AGAT</code></a>, <a href="https://github.com/gpertea/gffread" target="_blank"><code>GffRead</code></a>)</li>
  </ul>
</details>
<br>

<li><b>Mitogenomics</b></li>
  * Mitogenome assembly and annotation ([<code>MitoHiFi</code>](https://github.com/marcelauliano/MitoHiFi), [<code>MitoFinder</code>](https://github.com/RemiAllio/MitoFinder), [<code>mitoRNA</code>](https://github.com/RemiAllio/MitoFinder))

<li><b>Read manipulation</b></li>
  * Raw reads quality check ([<code>FastQC</code>](https://github.com/trinityrnaseq/trinityrnaseq/wiki), [<code>MultiQC</code>](https://multiqc.info/))
  * Read trimming ([<code>trimmomatic</code>](https://github.com/usadellab/Trimmomatic))
  * Read mapping and filtering ([<code>Bowtie2</code>](https://github.com/BenLangmead/bowtie2), [<code>STAR</code>](https://github.com/alexdobin/STAR), [<code>TEtranscripts</code>](https://github.com/mhammell-laboratory/TEtranscripts), [<code>stringtie</code>](https://github.com/gpertea/stringtie), [<code>samtools</code>](https://github.com/samtools/samtools))

<li><b>Transcriptomics</b></li>
  * De-novo transcriptome assembly ([<code>Trinity</code>](https://github.com/trinityrnaseq/trinityrnaseq/wiki))
  * Gene differential expression ([<code>NOISeq</code>](https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html), [<code>DESeq2</code>](https://bioconductor.org/packages/release/bioc/html/DESeq2.html), [<code>maSigPro</code>](https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html), [<code>topGO</code>](https://bioconductor.org/packages/release/bioc/html/topGO.html))

### 4.2. *Wet-lab*

* *Metaphase-plate chromosome preparation*
  * Experience with bivalves and ants
* *Total DNA extraction*
  * Experience with insects
* *Polymerase Chain Reaction (PCR)*
  * Experience with insect mitochondrial and nuclear markers
* *Mediterranean mussel (*Mytilus galloprovincialis*) embryology*
  * Spawning induction by thermal shock
  * Gamete collection and concentration
  * Sperm mitochondria staining with [MitoTracker](https://www.thermofisher.com/order/catalog/product/M7512)
  * *In-vitro* fertilization of eggs
  * Embryo rearing in flasks or multi-well plates
  * Estimation of survival rates
* *Hybridization Chain Reaction with RNA Fluorescence In-Situ Hybridization (HCR RNA-FISH)*
  * Experience with whole-mount embryos of the Mediterranean mussel (*Mytilus galloprovincialis*)
  * Probe design
  * Multiplex experiments (up to four target genes), paired with DAPI and MitoTracker stainings
  * Confocal imaging
  * Experience with Leica DMi8 paired with Stellaris 5 systems

### 4.3. *Others*
* *Field scientific sampling*
  * Experience with termites, ants, and freshwater crustaceans
  * Experience with mussels and oysters
  * Field sample storing for subsequent DNA extraction (pure methanol) or mRNA extraction (RNAlater or Trizol)
* *Scientific illustration* (see the [*Resources*](/resources/) section of this website)

## 5. Teaching experience

* **Practical class "CAFE: estimating gene family turover across a phylogenetic tree"**\
  *Master degree in Bioinformatics*\
  *Apr 23, 2024*
* **Practical invertebrate zoology class**\
  *Bachelor degree in Biological Sciences*\
  *Sep 2022 – Jan 2023*

## 6. Oral presentations at congresses

*For a report of poster presentations, please have a look also at the [Resources](/resources/) section in this website.*

* <ins>Nicolini F</ins>, Iannello M, Piccinini G, Ghiselli F, Luchetti A, Milani L. (2024). **Advancing the study of
bivalve sex determination in the light of comparative genomics**. *Establishing state-of-the-art mollusc genomics
(EMBO workshop)*. Namur (Belgium). May 27 – 30, 2024.
4

* <ins>Nicolini F</ins>, Ghiselli F, Milani L, Luchetti A. (2023). **Contrasting patterns of amino acid evolution
and shared ancestry between putative sex-determining genes in bivalve molluscs**. *EVOLMAR 2023*. Online. Nov 14 – 17, 2023.
* <ins>Nicolini F</ins>, Ghiselli F, Milani L, Luchetti A. (2023). **Sex-determination related genes in bivalves: novel acquisitions and high rates of sequence evolution**. *Evolution 2023* (*Ernst Mayr Award symposium*). Online. Jun 2 – 3, 2023.

## 7. Courses and workshops

* **Establishing state-of-the-art mollusc genomics**.\
  EMBO Workshop. Awarded with a travel grant.\
  *Namur, Belgium. May 28 – 31, 2024.*

* **Art (Science) Attack**.\
  Physalia Courses.\
  *Online. May 20 – 23, 2024.*

* **Introduction to Python for biologists**.\
  Physalia Courses.\
  *Online. Sep 25 – 28, 2023*

* **ITA\*PHY phylogenetics workshop**.\
  *Trento, Italy. Jun 6 – 9, 2023*

* **Sex chromosome evolution**.\
  Physalia Courses.\
  *Online. Jan 23 – 27, 2023*

* **Identification of butterflies from the Eastern Alps**.\
  Dolimiti National Park.\
  *Online. Mar 12 – Apr 2, 2022*

* **Science communication**.\
  Feltrinelli Education.\
  *Online. May 12 – Jun 3, 2021*

* **Nature illustration**.\
  Associazione ARDEA.\
  *Online. Feb 6 - Mar 20, 2021*

* **Advanced Open Water Diver (PADI Education)**.\
  Scientific Diving School (SDS).\
  *Bologna, Italy. Oct 18, 2020*

## 8. Awards and scholarships

* **Travel grant to attend the EMBO workshop "Establishing state-of-the-art mollusc genomics" in Namur (Belgium).**\
  EMBO.\
  *May 28 - 31, 2024*
* **Laura Bassi scholarship for editorial assistance to postgraduates and junior academics.**\
  Editing Press.\
  *Apr 13, 2023*

## 9. Visiting scholar

* **Nuzhdin's Lab**.\
University of Southern California. Los Angeles, California, USA.\
*Aug 20, 2023 – Feb 20, 2024*

* **Pasantes' Lab**.\
University of Vigo. Vigo, Spain.\
*Jan 12 – 22, 2023*


## 10. Society memberships

* **Italian Society for Evolutionary Biology** (SIBE/ISEB)\
  *Sep 2022 – ongoing*

* **Society for Molecular Biology and Evolution** (SMBE)\
  *Jul 2023 – ongoing*

* **Society for the Study of Evolution** (SSE)\
  *Jan 2023 – ongoing*

* **European Society for Evolutionary Developmental Biology** (EED)\
  *Apr 2022 – ongoing*