---
layout: archive
title: "Curriculum Vitae"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

<style>
summary {
  display: block;
}

/* Create a new custom triangle on the right side */
summary::before {
  margin-left: 1ch;
  display: inline-block;
  content: '+';
  transition: 0.2s;
}

details[open] > summary::before {
  transform: rotate(45deg);
}
</style>

<p><em>Last updated: Dec 2, 2024</em></p>

<p><em>Find the extended version of my CV in this <strong><a href="https://drive.google.com/file/d/1AtVbptyhfBHzxN-v-K8bItVa-xgiDXxv/view?usp=sharing" target="_blank">link</a></strong>.</em></p>

<h2>1. Professional experience</h2>
<p><strong>Research Associate</strong><br>
<strong>Subject</strong>: <em>Evolution of vision in early metazoans.</em><br>
<strong>Supervisor</strong>: <em>prof. Roberto Feuda</em><br>
<strong>University of Leicester, Leicester (UK). Nov 2024 - ongoing</strong></p>

<h2>2. Education</h2>

<p><strong>Ph.D</strong> in "Innovative technologies and sustainable use of Mediterranean Sea fishery and biological resources" (FishMed-PhD)<br>
<strong>Thesis</strong>: <em>Integrative perspectives on bivalve sex determination: A comparative and evolutionary analysis across phylogeny</em><br>
<strong>Supervisor</strong>: prof. Andrea Luchetti<br>
<strong>University of Bologna, Bologna (IT). Nov 2021 - Oct 2024</strong></p>

<p><strong>M.Sc</strong> in "Biodiversity and evolution"<br>
<strong>Thesis</strong>: <em>Identification of Hox and ParaHox genes in tadpole shrimp genomes (Pancrustacea, Branchiopoda, Notostraca)</em><br>
<strong>Supervisor</strong>: prof. Andrea Luchetti<br>
<strong>University of Bologna, Bologna (IT). Sep 2019 - Jul 2021</strong></p>

<p><strong>B.Sc</strong> in "Biological Sciences"<br>
<strong>Thesis</strong>: <em>Species delimitation and phylogenetic analysis of the Australian genus Candovia (Insecta, Phasmida)</em><br>
<strong>Supervisor</strong>: prof. Barbara Mantovani<br>
<strong>University of Bologna, Bologna (IT). Sep 2016 - Oct 2019</strong></p>

<h2>3. Publications</h2>

<p><ins>Nicolini F</ins>, Ghiselli F, Luchetti A, & Milani L. (2023). <strong>Bivalves as emerging model systems to study the mechanisms and evolution of sex determination: a genomic point of view</strong>. <em>Genome Biology and Evolution</em>, <em>15</em>(10), evad181. doi: <a href="https://doi.org/10.1093/gbe/evad181" target="_blank">10.1093/gbe/evad181</a></p>

<p>Martelossi J, <ins>Nicolini F</ins>, Subacchi S, Pasquale D, Ghiselli F, & Luchetti A. (2023). <strong>Multiple and diversified transposon lineages contribute to early and recent bivalve genome evolution</strong>. <em>BMC Biology</em>, <em>21</em>(1), 1-23. doi: <a href="https://doi.org/10.1186/s12915-023-01632-z" target="_blank">10.1186/s12915-023-01632-z</a></p>

<p><ins>Nicolini F</ins>, Martelossi J, Forni G, Savojardo C, Mantovani B, & Luchetti A. (2023). <strong>Comparative genomics of Hox and ParaHox genes among major lineages of Branchiopoda with emphasis on tadpole shrimps</strong>. <em>Frontiers in Ecology and Evolution</em>, 11, 23. doi: <a href="https://doi.org/10.3389/fevo.2023.1046960" target="_blank">10.3389/fevo.2023.1046960</a></p>

<p>Forni G, Cussigh A, Brock PD, Jones BR, <ins>Nicolini F</ins>, Martelossi J, ... & Mantovani B. (2023). <strong>Taxonomic revision of the Australian stick insect genus Candovia (Phasmida: Necrosciinae): insight from molecular systematics and species-delimitation approaches</strong>. <em>Zoological Journal of the Linnean Society</em>, <em>197</em>(1), 189-210. doi: <a href="https://doi.org/10.1093/zoolinnean/zlac074" target="_blank">10.1093/zoolinnean/zlac074</a></p>

<h2>4. Outreach activity</h2>

<p><em>For a more detailed description of my outreach commitment, please have a look also at the <a href="/outreach/">Outreach</a> section in this website.</em></p>

<ul>
  <li><strong>Editor and web writer</strong><br>BioPills. Jul 2017 – ongoing</li>
  <li><strong>BiGeA Day 2023</strong><br>University of Bologna, Bologna (IT). May 27, 2023</li>
  <li><strong>University of Bologna opening days</strong><br>University of Bologna, Bologna (IT). Nov 18, 2022</li>
</ul>

<h2>5. Professional skills</h2>

<h3>5.1. Bioinformatics</h3>

<ul>
  <li><b>Programming and markup languages</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <small>(beginner <i class="fa-regular fa-star"></i><i class="fa-regular fa-star" style="color: #494E52;"></i><i class="fa-regular fa-star" style="color: #494E52;"></i><i class="fa-light fa-star" style="color: #494E52;"></i><i class="fa-light fa-star" style="color: #494E52;"></i> advanced)</small>
      <ul>
        <li><code>bash</code><br /> <i class="fa-solid fa-star" style="color: #494E52;"></i>▮▮▮▮▯</li>
        <li><code>R</code><br /> ▮▮▮▯▯</li>
        <li><code>python</code><br /> ▮▮▯▯▯</li>
        <li>LaTeX<br /> ▮▮▯▯▯</li>
        <li><code>html</code><br /> ▮▯▯▯▯</li>
      </ul>
    </details>
  </li>

  <li><b>Phylogenetics</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>ML phylogenetic analysis (<a href="https://github.com/iqtree/iqtree2" target="_blank"><code>IQTREE</code></a>, <a href="https://github.com/amkozlov/raxml-ng" target="_blank"><code>RAxML-NG</code></a>, <a href="http://www.microbesonline.org/fasttree/" target="_blank"><code>FastTree</code></a>)</li>
        <li>Bayesian phylogenetic analysis (<a href="https://nbisweden.github.io/MrBayes/" target="_blank"><code>MrBayes</code></a>, <a href="http://www.atgc-montpellier.fr/phylobayes/" target="_blank"><code>PhyloBayes</code></a>)</li>
        <li>Global alignment tools (<a href="https://mafft.cbrc.jp/alignment/software/" target="_blank"><code>MAFFT</code></a>, <a href="https://github.com/rcedgar/muscle" target="_blank"><code>MUSCLE</code></a>, <a href="http://www.clustal.org/omega/" target="_blank"><code>Clustal Omega</code></a>, <a href="http://trimal.cgenomics.org/" target="_blank"><code>trimAl</code></a>, <a href="https://home.cc.umanitoba.ca/~psgendb/doc/Castresana/Gblocks_documentation.html" target="_blank"><code>GBlocks</code></a>)</li>
        <li>Tree calibration (<a href="https://github.com/tothuhien/lsd2" target="_blank"><code>lsd2</code></a>, <a href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank"><code>MCMCTree</code></a>)</li>
        <li>Tree decomposition (<a href="https://github.com/JSdoubleL/DISCO" target="_blank"><code>DISCO</code></a>)</li>
      </ul>
    </details>
  </li>

  <li><b>Comparative genomics</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Orthology inference (<a href="https://github.com/davidemms/OrthoFinder" target="_blank"><code>OrthoFinder</code></a>, <a href="https://github.com/xgrau/possvm-orthology" target="_blank"><code>Possvm</code></a>, <a href="https://github.com/hillerlab/TOGA" target="_blank"><code>TOGA</code></a>)</li>
        <li>Gene annotation (<a href="https://github.com/ncbi/blast_plus_docs" target="_blank"><code>BLAST</code></a>, <a href="https://github.com/bbuchfink/diamond" target="_blank"><code>DIAMOND</code></a>, <a href="https://github.com/EddyRivasLab/hmmer" target="_blank"><code>HMMER</code></a>, <a href="https://github.com/ebi-pf-team/interproscan" target="_blank"><code>InterProScan</code></a>, <a href="https://github.com/UCSC-LoweLab/tRNAscan-SE" target="_blank"><code>tRNAscan-SE</code></a>)</li>
        <li>Genome quality assessment (<a href="https://busco.ezlab.org/" target="_blank"><code>BUSCO</code></a>)</li>
        <li>Molecular evolution (<a href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank"><code>PAML</code></a>, <a href="http://hyphy.org/" target="_blank"><code>HyPhy</code></a>, <a href="https://github.com/for-giobbe/BASE" target="_blank"><code>BASE</code></a>, <a href="https://github.com/mrrlab/RRTree" target="_blank"><code>RRTree</code></a>)</li>
        <li>Gene family evolution (<a href="https://github.com/hahnlab/CAFE5" target="_blank"><code>CAFE5</code></a>, <a href="https://github.com/BenoitMorel/GeneRax" target="_blank"><code>GeneRax</code></a>)</li>
        <li>Gene macro-synteny (<a href="https://github.com/wyp1125/MCScanX" target="_blank"><code>MCScanX</code></a>/<a href="https://github.com/tanghaibao/jcvi" target="_blank"><code>JCVI</code></a>)</li>
        <li>Detection of sex-related loci (<a href="https://gitlab.in2p3.fr/sex-det-family/sex-detector" target="_blank"><code>SEX-DETector</code></a>, <a href="https://gitlab.in2p3.fr/sex-det-family/sdpop" target="_blank"><code>SDpop</code></a>)</li>
        <li>Other (<a href="https://github.com/NBISweden/AGAT" target="_blank"><code>AGAT</code></a>, <a href="https://github.com/gpertea/gffread" target="_blank"><code>GffRead</code></a>)</li>
      </ul>
    </details>
  </li>

  <li><b>Mitogenomics</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Mitogenome assembly and annotation (<a href="https://github.com/marcelauliano/MitoHiFi" target="_blank"><code>MitoHiFi</code></a>, <a href="https://github.com/RemiAllio/MitoFinder" target="_blank"><code>MitoFinder</code></a>, <a href="https://github.com/RemiAllio/MitoFinder" target="_blank"><code>mitoRNA</code></a>)</li>
      </ul>
    </details>
  </li>

  <li><b>Read manipulation</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Raw reads quality check (<a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki" target="_blank"><code>FastQC</code></a>, <a href="https://multiqc.info/" target="_blank"><code>MultiQC</code></a>)</li>
        <li>Read trimming (<a href="https://github.com/usadellab/Trimmomatic" target="_blank"><code>trimmomatic</code></a>)</li>
        <li>Read mapping and filtering (<a href="https://github.com/BenLangmead/bowtie2" target="_blank"><code>Bowtie2</code></a>, <a href="https://github.com/alexdobin/STAR" target="_blank"><code>STAR</code></a>, <a href="https://github.com/mhammell-laboratory/TEtranscripts" target="_blank"><code>TEtranscripts</code></a>, <a href="https://github.com/gpertea/stringtie" target="_blank"><code>stringtie</code></a>, <a href="https://github.com/samtools/samtools" target="_blank"><code>samtools</code></a>)</li>
      </ul>
    </details>
  </li>

  <li><b>Transcriptomics</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>De-novo transcriptome assembly (<a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki" target="_blank"><code>Trinity</code></a>)</li>
        <li>Gene differential expression (<a href="https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html" target="_blank"><code>NOISeq</code></a>, <a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" target="_blank"><code>DESeq2</code></a>, <a href="https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html" target="_blank"><code>maSigPro</code></a>, <a href="https://bioconductor.org/packages/release/bioc/html/topGO.html" target="_blank"><code>topGO</code></a>)</li>
      </ul>
    </details>
  </li>

</ul>

<h3>5.2. <i>Wet-lab</i></h3>

<ul>
  <li><b>Metaphase-plate chromosome preparation</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Experience with bivalves and ants</li>
      </ul>
    </details>
  </li>

  <li><b>Total DNA extraction</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Experience with insects</li>
      </ul>
    </details>
  </li>

  <li><b>Polymerase Chain Reaction (PCR)</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Experience with insect mitochondrial and nuclear markers</li>
      </ul>
    </details>
  </li>

  <li><b>Mediterranean mussel (<i>Mytilus galloprovincialis</i>) embryology</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Spawning induction by thermal shock</li>
        <li>Gamete collection and concentration</li>
        <li>Sperm mitochondria staining with <a href="https://www.thermofisher.com/order/catalog/product/M7512" target="_blank">MitoTracker</a></li>
        <li><i>In-vitro</i> fertilization of eggs</li>
        <li>Embryo rearing in flasks or multi-well plates</li>
        <li>Estimation of survival rates</li>
      </ul>
    </details>
  </li>

  <li><b>mRNA <i>in-situ</i> Hybridization Chain Reaction (mRNA <i>in-situ</i> HCR)</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Experience with whole-mount embryos of the Mediterranean mussel (<i>Mytilus galloprovincialis</i>)</li>
        <li>Probe design</li>
        <li>Multiplex experiments (up to four target genes), paired with DAPI and MitoTracker stainings</li>
        <li>Confocal imaging (main experience with Leica DMi8 paired with Stellaris 5 systems)</li>
      </ul>
    </details>
  </li>

</ul>

<h3>5.3. Others</h3>

<ul>
  <li><b>Field scientific sampling</b>
    <details>
      <summary><i><small> More</small></i></summary>
      <ul>
        <li>Experience with termites, ants, and freshwater crustaceans</li>
        <li>Experience with mussels and oysters</li>
        <li>Field sample storing for subsequent DNA extraction (pure methanol) or mRNA extraction (RNAlater or Trizol)</li>
      </ul>
    </details>
  </li>

  <li><b>Scientific illustration</b><br>
    <small><i>See the <a href="/resources/"><b>Resources</b></a> section of this website</i></small>
  </li>

</ul>

<h2>6. Teaching experience</h2>

<ul>
  <li><b>Practical class "CAFE: estimating gene family turnover across a phylogenetic tree"</b><br />
    Master degree in <i>Bioinformatics</i><br />
    <i>University of Bologna, Bologna (IT). Apr 23, 2024</i></li>
  <li><b>Practical invertebrate zoology class</b><br />
    Bachelor degree in <i>Biological Sciences</i><br />
    <i>University of Bologna, Bologna (IT). Sep 2022 – Jan 2023</i></li>
</ul>

<h2>7. Oral presentations at congresses</h2>

<p><i>For a report of poster presentations, please have a look also at the <a href="/resources/"><b>Resources</b></a> section of this website.</i></p>

<ul>
  <li><ins>Nicolini F</ins>, Iannello M, Piccinini G, Ghiselli F, Luchetti A, Milani L. (2024). <b>Advancing the study of bivalve sex determination in the light of comparative genomics</b>. <i>Establishing state-of-the-art mollusc genomics (EMBO workshop)</i>. Namur (BE). May 27 – 30, 2024.</li>
  <li><ins>Nicolini F</ins>, Ghiselli F, Milani L, Luchetti A. (2023). <b>Contrasting patterns of amino acid evolution and shared ancestry between putative sex-determining genes in bivalve molluscs</b>. <i>EVOLMAR 2023</i>. Online. Nov 14 – 17, 2023.</li>
  <li><ins>Nicolini F</ins>, Ghiselli F, Milani L, Luchetti A. (2023). <b>Sex-determination related genes in bivalves: novel acquisitions and high rates of sequence evolution</b>. <i>Evolution 2023</i> (<i>Ernst Mayr Award symposium</i>). Online. Jun 2 – 3, 2023.</li>
</ul>

<h2>8. Courses and workshops</h2>

<ul>
  <li><b>Establishing state-of-the-art mollusc genomics</b><br />
    EMBO Workshop. Awarded with a travel grant.<br />
    <i>Namur, Belgium. May 28 – 31, 2024.</i></li>

  <li><b>Art (Science) Attack</b><br />
    Physalia Courses.<br />
    <i>Online. May 20 – 23, 2024.</i></li>

  <li><b>Introduction to Python for biologists</b><br />
    Physalia Courses.<br />
    <i>Online. Sep 25 – 28, 2023</i></li>

  <li><b>ITA*PHY phylogenetics workshop</b><br />
    <i>Trento (IT). Jun 6 – 9, 2023</i></li>

  <li><b>Sex chromosome evolution</b><br />
    Physalia Courses.<br />
    <i>Online. Jan 23 – 27, 2023</i></li>

  <li><b>Identification of butterflies from the Eastern Alps</b><br />
    Dolimiti National Park.<br />
    <i>Online. Mar 12 – Apr 2, 2022</i></li>

  <li><b>Science communication</b><br />
    Feltrinelli Education.<br />
    <i>Online. May 12 – Jun 3, 2021</i></li>

  <li><b>Nature illustration</b><br />
    Associazione ARDEA.<br />
    <i>Online. Feb 6 – Mar 20, 2021</i></li>

  <li><b>Advanced Open Water Diver (PADI Education)</b><br />
    Scientific Diving School (SDS).<br />
    <i>Bologna (IT). Oct 18, 2020</i></li>
</ul>

<h2>9. Awards and scholarships</h2>

<ul>
  <li><b>Travel grant to attend the EMBO workshop "Establishing state-of-the-art mollusc genomics" in Namur (Belgium).</b><br />
    EMBO.<br />
    <i>May 28 – 31, 2024</i></li>

  <li><b>Laura Bassi scholarship for editorial assistance to postgraduates and junior academics.</b><br />
    Editing Press.<br />
    <i>Apr 13, 2023</i></li>
</ul>

<h2>10. Visiting scholar</h2>

<ul>
  <li><b>Nuzhdin's Lab</b><br />
    <i>University of Southern California, Los Angeles (CA, USA). Aug 2023 – Feb 2024</i></li>

  <li><b>Pasantes' Lab</b><br />
    <i>University of Vigo, Vigo (ES). Jan 12 – 22, 2023</i></li>
</ul>

<h2>11. Society memberships</h2>

<ul>
  <li><b>Italian Society for Evolutionary Biology</b> (SIBE/ISEB)<br />
    <i>Sep 2022 – ongoing</i></li>

  <li><b>Society for Molecular Biology and Evolution</b> (SMBE)<br />
    <i>Jul 2023 – ongoing</i></li>

  <li><b>Society for the Study of Evolution</b> (SSE)<br />
    <i>Jan 2023 – ongoing</i></li>

  <li><b>European Society for Evolutionary Developmental Biology</b> (EED)<br />
    <i>Apr 2022 – ongoing</i></li>
</ul>


