---
layout: archive
title: "Curriculum Vitae"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

*Last updated: Nov 11, 2023*

*Please find an extended version of this CV by clicking this **<a href="https://drive.google.com/file/d/1e2bsp4sTa-pi7TeGBGKfEYRMSBEX6SFw/view?usp=sharing" target="_blank">link</a>**.*

## 1. Education

* **Ph.D** in "Innovative technologies and sustainable use of Mediterranean Sea fishery and biological resources" (FishMed-PhD), University of Bologna, 2025 (expected)\
*Subject: Comparative genomics of bivalve sex-determination realted genes*\
*Supervisor: prof. Andrea Luchetti*
* **M.Sc** in "Biodiversity and evolution", University of Bologna, 2021\
*Thesis: Identification of* Hox *and* ParaHox *genes in tadpole shrimp genomes (Pancrustacea, Branchiopoda, Notostraca)*\
*Supervisor: prof. Andrea Luchetti*
* **B.Sc** in "Biological Sciences", University of Bologna, 2019\
*Thesis: Species delimitation and phylogenetic analysis of the Australian genus* Candovia *(Insecta, Phasmida)*\
*Supervisor: prof. Barbara Mantovani*

## 2. Publications

* <ins>Nicolini F</ins>, Ghiselli F, Luchetti A, & Milani L. (2023). **Bivalves as emerging model systems to study the mechanisms and evolution of sex determination: a genomic point of view**. *Genome Biology and Evolution*, *15*(10), evad181. doi: https://doi.org/10.1093/gbe/evad181

* Martelossi J, <ins>Nicolini F</ins>, Subacchi S, Pasquale D, Ghiselli F, & Luchetti A. (2023). **Multiple and diversified transposon lineages contribute to early and recent bivalve genome evolution**. *BMC Biology*, *21*(1), 1-23. doi: https://doi.org/10.1186/s12915-023-01632-z

* <ins>Nicolini F</ins>, Martelossi J, Forni G, Savojardo C, Mantovani B, & Luchetti A. (2023). **Comparative genomics of *Hox* and *ParaHox* genes among major lineages of Branchiopoda with emphasis on tadpole shrimps**. *Frontiers in Ecology and Evolution*, 11, 23. doi: https://doi.org/10.3389/fevo.2023.1046960

* Forni G, Cussigh A, Brock PD, Jones BR, <ins>Nicolini F</ins>, Martelossi J, ... & Mantovani B. (2023). **Taxonomic revision of the Australian stick insect genus *Candovia* (Phasmida: Necrosciinae): insight from molecular systematics and species-delimitation approaches**. *Zoological Journal of the Linnean Society*, *197*(1), 189-210. doi: https://doi.org/10.1093/zoolinnean/zlac074

## 3. Outreach activity

*Have also a look at the [Outreach](/outreach/) section*
* Editor and web writer, BioPills\
  *Jul 2017 – ongoing*

* BiGeA Day 2023, University of Bologna\
  *May 27, 2023*

* University of Bologna opening days\
  *Nov 18, 2022*

## 4. Professional skills

### 4.1. Bioinformatics

* *Programming and command languages* (beginner ∘∘∘∘∘ advantaced)
  * <code>bash</code>\
    ●●●●∘
  * <code>R</code>\
    ●●●∘∘
  * <code>python</code>\
    ●●∘∘∘
  * $\LaTeX$\
    ●∘∘∘∘
  * <code>html</code>\
    ●∘∘∘∘


* *Phylogenetics*
  * ML phylogenetic analysis ([<code>IQTREE</code>](https://github.com/iqtree/iqtree2), [<code>RAxML-NG</code>](https://github.com/amkozlov/raxml-ng), [<code>FastTree</code>](http://www.microbesonline.org/fasttree/))
  * Bayesian phylogenetic analysis ([<code>MrBayes</code>](https://nbisweden.github.io/MrBayes/), [<code>PhyloBayes</code>](http://www.atgc-montpellier.fr/phylobayes/))
  * Global alignment tools ([<code>MAFFT</code>](https://mafft.cbrc.jp/alignment/software/), [<code>MUSCLE</code>](https://github.com/rcedgar/muscle), [<code>Clustal Omega</code>](http://www.clustal.org/omega/), [<code>trimAl</code>](http://trimal.cgenomics.org/), [<code>GBlocks</code>](https://home.cc.umanitoba.ca/~psgendb/doc/Castresana/Gblocks_documentation.html))
  * Tree calibration ([<code>lsd2</code>](https://github.com/tothuhien/lsd2), [<code>MCMCTree</code>](http://abacus.gene.ucl.ac.uk/software/paml.html))
  * Tree decomposition ([<code>DISCO</code>](https://github.com/JSdoubleL/DISCO))

* *Comparative genomics*
  * Orthology inference ([<code>OrthoFinder</code>](https://github.com/davidemms/OrthoFinder), [<code>Possvm</code>](https://github.com/xgrau/possvm-orthology), [<code>TOGA</code>](https://github.com/hillerlab/TOGA))
  * Gene annotation ([<code>BLAST</code>](https://github.com/ncbi/blast_plus_docs), [<code>DIAMOND</code>](https://github.com/bbuchfink/diamond), [<code>HMMER</code>](https://github.com/EddyRivasLab/hmmer), [<code>InterProScan</code>](https://github.com/ebi-pf-team/interproscan), [<code>tRNAscan-SE</code>](https://github.com/UCSC-LoweLab/tRNAscan-SE))
  * Genome quality assessment ([<code>BUSCO</code>](https://busco.ezlab.org/))
  * Molecular evolution ([<code>PAML</code>](http://abacus.gene.ucl.ac.uk/software/paml.html), [<code>HyPhy</code>](http://hyphy.org/), [<code>BASE</code>](https://github.com/for-giobbe/BASE), [<code>RRTree</code>](https://github.com/mrrlab/RRTree))
  * Gene macro-synteny ([<code>MCScanX</code>](https://github.com/wyp1125/MCScanX)/[<code>JCVI</code>](https://github.com/tanghaibao/jcvi))
  * Detection of sex-related loci ([<code>SEX-DETector</code>](https://gitlab.in2p3.fr/sex-det-family/sex-detector), [<code>SDpop</code>](https://gitlab.in2p3.fr/sex-det-family/sdpop))
  * Other ([<code>AGAT</code>](https://github.com/NBISweden/AGAT), [<code>GffRead</code>](https://github.com/gpertea/gffread))

* *Mitogenomics*
  * Mitogenome assembly and annotation ([<code>MitoHiFi</code>](https://github.com/marcelauliano/MitoHiFi), [<code>MitoFinder</code>](https://github.com/RemiAllio/MitoFinder), [<code>mitoRNA</code>](https://github.com/RemiAllio/MitoFinder))

* *Read manipulation*
  * Raw reads quality check ([<code>FastQC</code>](https://github.com/trinityrnaseq/trinityrnaseq/wiki), [<code>MultiQC</code>](https://multiqc.info/))
  * Read trimming ([<code>trimmomatic</code>](https://github.com/usadellab/Trimmomatic))
  * Read mapping and filtering ([<code>Bowtie2</code>](https://github.com/BenLangmead/bowtie2), [<code>STAR</code>](https://github.com/alexdobin/STAR), [<code>TEtranscripts</code>](https://github.com/mhammell-laboratory/TEtranscripts), [<code>samtools</code>](https://github.com/samtools/samtools))

* *Transcriptomics*
  * De-novo transcriptome assembly ([<code>Trinity</code>](https://github.com/trinityrnaseq/trinityrnaseq/wiki))
  * Gene differential expression ([<code>NOISeq</code>](https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html), [<code>DESeq2</code>](https://bioconductor.org/packages/release/bioc/html/DESeq2.html), [<code>maSigPro</code>](https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html), [<code>topGO</code>](https://bioconductor.org/packages/release/bioc/html/topGO.html))

### 4.2. *Wet-lab*

* Metaphase-plate chromosome preparation
* Total DNA extraction
* PCR

## 5. Teaching assistant

* Practical invertebrate zoology class\
  *Sep 2022 – Jan 2023*

## 6. Presentations at congresses

*For a collection of poster presentations please visit the [Other resources](/resources/) section*

* <ins>Nicolini, F.</ins>, Ghiselli, F., Milani, L., Luchetti, A. (2023). **Sex-determination related genes in bivalves: novel acquisitions and high rates of sequence evolution**. *Evolution 2023* (*Ernst Mayr Award symposium*). Online. Jun 2 – 3, 2023.

## 7. Courses and workshops

* **Introduction to Python for biologists**.\
  Physalia Courses.\
  *Online. Sep 25 – 28, 2023*

* **ITA\*PHY phylogenetics workshop**.\
  *Trento, Italy. Jun 6 – 9, 2023*

* **Sex chromosome evolution**.\
  Physalia Courses.\
  *Online. Jan 23 – 27, 2023*

* **Identification of butterflies from the Eastern Alps**.\
  Dolimiti National Park.\
  *Online. Mar 12 – Apr 2, 2022*

* **Science communication**.\
  Feltrinelli Education.\
  *Online. May 12 – Jun 3, 2021*

* **Nature illustration**.\
  Associazione ARDEA.\
  *Online. Feb 6 - Mar 20, 2021*

## 8. Awards and scholarships

* **Laura Bassi scholarship for editorial assistance to postgraduates and junior academics.**\
  Editing Press.\
  *Apr 13, 2023*

## 9. Visiting fellow

* **Sergey Nuzhdin's group**.\
University of Southern California. Los Angeles, California, USA.\
*Aug 20, 2023 – ongoing*

* **Juan Pasantes' group**.\
University of Vigo. Vigo, Spain.\
*Jan 12 – 22, 2023*


## 10. Society memberships

* **Italian Society for Evolutionary Biology** (SIBE/ISEB)\
  *Sep 2022 – ongoing*

* **Society for Molecular Biology and Evolution** (SMBE)\
  *Jul 2023 – ongoing*

* **Society for the Study of Evolution** (SSE)\
  *Jan 2023 – ongoing*

* **European Society for Evolutionary Developmental Biology** (EED)\
  *Apr 2022 – ongoing*